app/assets/javascripts/application/thumbnail_live_update.js

```diff
@@ -1,20 +1,44 @@
-module("thumbnail", function (module) {
-  module.exports = (function(){
+var thumbnailLiveUpdate = function(){
+  var init = function(){
+    _updateTitle();
+    _updateDate();
+    _updatePrice();
+    _updateShortDescription();
+    _updateLocation();
+  };
+
+  var _updateTitle = function(){
     $(".event_title input").focusout(function (e) {
       $(".event-thumbnail h2").text($(this).val());
-    })
+    });
+  };
+
+  var _updateDate = function(){
     $(".event_date input").focusout(function (e) {
       var datetime = moment(new Date($(this).val())).format("Do MMMM YYYY h:mma")
       $(".event-thumbnail span.date").text(datetime);
-    })
+    });
+  };
+
+  var _updatePrice = function(){
     $(".event_price input").focusout(function (e) {
       $(".event-thumbnail span.price").text("£"+$(this).val());
-    })
+    });
+  };
+
+  var _updateShortDescription = function(){
     $(".event_short_description input").focusout(function (e) {
       $(".event-thumbnail .short-description").text($(this).val());
-    })
+    });
+  };
+
+  var _updateLocation = function(){
     $(".event_location input").focusout(function (e) {
       $(".event-thumbnail .venue").text($(this).val());
-    })
-  })();
-});
+    });
+  };
+
+  return {
+    init: init
+  };
+}();
```

Comment:
Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.